mmv1/templates/terraform/examples/app_engine_flexible_app_version.tf.erb

```diff
@@ -71,6 +77,7 @@ resource "google_app_engine_flexible_app_version" "<%= ctx[:primary_resource_id]
   }
 
   noop_on_destroy = true
+  service_account = google_service_account.custom_service_account.id
```

Comment:
It looks like the current error is back to:

```
"The user does not have access to service account \'One of [projects/tf-test-appeng-flexj81z9fknvo/serviceAccounts/tf-test-my-accountj81z9fknvo@tf-test-appeng-flexj81z9fknvo.iam.gserviceaccount.com]\'.  User: \'697236229335@cloudservices.gserviceaccount.com\'.  Ask a project owner to grant you the iam.serviceAccountUser role on the service account\"
```

The id is the full resource id, for example: `projects/tf-test-appeng-flexj81z9fknvo/serviceAccounts/tf-test-my-accountj81z9fknvo@tf-test-appeng-flexj81z9fknvo.iam.gserviceaccount.com`. The API docs are a little ambiguous - possibly this should be just the email?

```suggestion
  service_account = google_service_account.custom_service_account.email
```

But I think you had said you were able to run this test successfully locally - is that correct? If you can, it's probably a real issue with the configuration of the test project's IAM policies.

Reply:
I see. just providing email should work.
no, i have moved to a new lappy. There are some issues with local setup now. Will check locally if the issue persists.